src/sentry/eventstream/kafka/backend.py

```diff
@@ -69,6 +71,9 @@ def encode_bool(value: Optional[bool]) -> str:
                 value = False
             return str(int(value))
 
+        def encode_dict(value: Mapping) -> str:
+            return json.dumps(value)
```

Comment:
Not sure if we want to encode the dict in a more optimal format.

Reply:
I think this should be ok. I guess since these are mostly bools we could optimize if necessary